crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -1158,6 +1145,36 @@ fn create_inventory_fragments_message(
 }
 
 impl CumulocityConverter {
+    /// Register on C8y an operation capability for a device.
+    fn register_operation(
+        &self,
+        device: &EntityMetadata,
+        c8y_operation_name: &str,
```

Comment:
I don't get your comment. This type is here to *hold* an actual operation request, not to declare an operation capability. Am I missing something?

Reply:
Oops sorry. I thought we had an enum with all the supported operation variants listed, but unfortunately no. We probably should still have one in the `smartrest` module to avoid passing operation names as `&str`s like this.

Reply:
Actually there is one: `CumulocitySupportedOperations`.